website/content/docs/setup/quickstart/index.md

```diff
@@ -7,7 +7,7 @@ This guide lets you quickly evaluate {{< name >}}. For a custom setup follow the
 
 ## Before you begin
 Ensure that you have the following:
-- `kubectl` with access to a Kubernetes cluster (this guide was tested with kind v0.10.0 and OpenShift 4.3)
```

Comment:
kubectl is used in the text below, it is a dependency.

Reply:
ok, I have fixed